sinks/gcm/driver.go

```diff
@@ -366,27 +395,23 @@ func fullLabelName(name string) string {
 	return name
 }
 
-func fullMetricName(name string, metricType sink_api.MetricType) string {
-	// Suffix cumulative metrics with "_delta" since we're changing them to gauges.
-	// This will ease the transition to cumulative metrics when those come.
-	suffix := ""
-	if metricType == sink_api.MetricCumulative {
-		suffix = "_delta"
-	}
-
-	if !strings.Contains(name, "custom.cloudmonitoring.googleapis.com/") {
-		return fmt.Sprintf("custom.cloudmonitoring.googleapis.com/%s/%s%s", metricDomain, name, suffix)
+func fullMetricName(name string) string {
+	if !strings.Contains(name, customApiPrefix) {
```

Comment:
Shouldn't we not need to do this btw? We should know if the metrics coming in are ours or not no?